lib/Items/items.cpp

```diff
@@ -22,3 +22,8 @@ Item::Item(ItemType type, int x, int y, int size)
   }
 };
 
+Item::~Item()
+{
+  size = 0;
+}
+
```

Comment:
in a deconstructor use the delete functions : https://learn.microsoft.com/en-us/cpp/cpp/destructors-cpp?view=msvc-170

Reply:
dan doet ie boos :(